.pre-commit-config.yaml

```
@@ -28,3 +35,13 @@ repos:
     rev: v3.0.3
     hooks:
       - id: prettier
+        stages:
+          - pre-commit
+  - repo: https://github.com/commitizen-tools/commitizen
+    rev: v3.15.0 # automatically updated by Commitizen
```

```suggestion
    rev: v3.15.0
```
我们这边会不定期pre-commit update的，平常保证ci版本不变